### ISSUE TITLE
Where is 3D convolution？

### ISSUE THREAD

**Issue by liuky74** [mindspore-ai/mindspore#12]
I took a look at the documentation and didn't find any related classes for 3D operation. Isn't there yet?

**Comment by zhangjerry27** [comment]
Yes, 3D convolution operation isn't supported yet, we're going to support it later.

**Comment by sliawatimena** [comment]
Is there any explanation about 3D Convolution operation? Thanks.
